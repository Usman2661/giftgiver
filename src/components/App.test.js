import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

const app = shallow(<App />);

it('Renders the app correctly' , () =>{
    expect(app).toMatchSnapshot();
});