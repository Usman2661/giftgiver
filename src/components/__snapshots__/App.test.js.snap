// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders the app correctly 1`] = `
<div>
  <h2>
    Gift Giver 
  </h2>
  <div
    className="gift-list"
  />
  <Button
    active={false}
    className="btn-add"
    disabled={false}
    onClick={[Function]}
    variant="primary"
  >
     Add Gift
  </Button>
</div>
`;
